db/migrate/20211103112944_translate_budget_phase_main_link_url.rb

```diff
@@ -0,0 +1,6 @@
+class TranslateBudgetPhaseMainLinkUrl < ActiveRecord::Migration[5.2]
+  def change
+    add_column :budget_phase_translations, :main_link_url, :string
+    remove_column :budget_phases, :main_link_url
```

Comment:
```suggestion
    remove_column :budget_phases, :main_link_url, :string
```

This way we make the migration reversible :wink:.

Reply:
Done!